CHANGELOG.rst

```diff
@@ -38,11 +44,22 @@ User-Facing Changes
   `#2357 <https://github.com/SwissDataScienceCenter/renku-ui/pull/2357>`_).
 * **UI**: Sort commits by date to prevent random order
   (`#2347 <https://github.com/SwissDataScienceCenter/renku-ui/pull/2347>`_).
+* **Gateway**: Use offline access tokens for automated access from within sessions.
+  (`#632 <https://github.com/SwissDataScienceCenter/renku-gateway/pull/632>`_).
+* **Sessions**: Propagate environment variables for R-Studio sessions
+  (`#1339 <https://github.com/SwissDataScienceCenter/renku-notebooks/pull/1339>`_).
+* **Sessions**: Show if ssh is enabled in /version of notebook service
+  (`#1407 <https://github.com/SwissDataScienceCenter/renku-notebooks/pull/1407>`_).
+* **Sessions**: Cloning the correct SHA for anonymous user sessions
+  (`#1406 <https://github.com/SwissDataScienceCenter/renku-notebooks/pull/1406>`_).
```

Comment:
Should these go under the "Internal changes" section, instead of the "User Facing Changes" section?

```suggestion

Internal Changes
~~~~~~~~~~~~~~~~

**Improvements**

* **Sessions**: Show if ssh is enabled in /version of notebook service
  (`#1407 <https://github.com/SwissDataScienceCenter/renku-notebooks/pull/1407>`_).

**Bug Fixes**

* **Sessions**: Cloning the correct SHA for anonymous user sessions
  (`#1406 <https://github.com/SwissDataScienceCenter/renku-notebooks/pull/1406>`_).
```